src/Mod/Sketcher/Gui/TaskSketcherConstraints.cpp

```diff
@@ -1483,11 +1503,74 @@ void TaskSketcherConstraints::changeEvent(QEvent *e)
     }
 }
 
-template <class T>
-bool TaskSketcherConstraints::isFilter(T filterValue) {
-    return isFilterMatch(filterValue, ui->comboBoxFilter->currentIndex());
-}
+/* list for multi filter */
+void TaskSketcherConstraints::on_filterList_itemChanged(QListWidgetItem* item)
+{
+    int filterindex = filterList->row(item);
+
+    auto itemAggregate = filterAggregates[filterindex];

```

Comment:
by adding multiple Base::Console().Warning calls and testing i determined that this is the line that makes freecad crash for me, I do not really understand what this does but I noticed there seems to be no entry for selection or associated filters in the declaration of filterAggregates

Reply:
Ok, I will move this to fix the issue.